core/cap-0059.md

```diff
@@ -50,7 +50,7 @@ This CAP is aligned with the following Stellar Network Goals:
         }
     ],
     "return": "BytesObject",
-    "docs": "Adds two BLS12-381 G1 points given in bytes format and returns the resulting G1 point in bytes format.",
+    "docs": "Adds two BLS12-381 G1 points given in bytes format and returns the resulting G1 point in bytes format. G1 serialization format: `be_encode(X) || be_encode(Y)` and the most significant three bits of X encodes flags, i.e.  bits(X) = [compression_flag, infinity_flag, sort_flag, bit_380, .. bit_0]",
```

Comment:
How about mentioning big endian like we have in some changes below? be_encode might not be intuitive for everyone.